src/luminator.ts

```diff
@@ -40,33 +46,41 @@ class Luminator {
   private proxyManagerOptions: IProxyManagerOption;
   private sessionId: number;
 
-  constructor(username: string, password: string, config: ILuminatorConfig = Luminator.DEFAULT_CONFIG) {
+  constructor(
+    username: string,
+    password: string,
+    config: ILuminatorConfig = Luminator.DEFAULT_CONFIG,
+    axiosDefaultConfig: AxiosRequestConfig = {},
+  ) {
     this.username = username;
     this.password = password;
     this.superProxy = config.superProxy;
     this.country = config.country;
     this.port = config.port;
+    this.axiosDefaultConfig = axiosDefaultConfig;
   }
 
   /**
    * @description Generate a random id.
    * @return {number}
    */
   private static getSessionId(): number {
-    return Math.trunc(crypto.randomBytes(8).readUInt32LE(1) / 0xffffffff * 1000000);
+    return Math.trunc(
+      (crypto.randomBytes(8).readUInt32LE(1) / 0xffffffff) * 1000000,
+    );
   }
 
   /**
    * @description return status if it is a valid AxiosError
```

Comment:
* @description return status if it is a valid AxiosError

to

 * @description Return status if it is a valid AxiosError